crates/solver/src/liquidity/order_converter.rs

```diff
@@ -66,7 +66,10 @@ impl OrderConverter {
 
         // The reported fee amount that is used for objective computation is the
         // order's full full amount scaled by a constant factor.
-        let solver_fee = remaining.remaining(order.metadata.solver_fee)?;
+        let solver_fee = match order.solver_determines_fee() {
```

Comment:
While this is fine for an immediate hotfix, we want to not compute these fees in the first place (as they are costing us API requests), so the change should likely be done in the autopilot where it is computed.

Reply:
Yes, all `surplus_fee` calculation will be cleanup once we make sure everything works as expected.